apps/devhub/views.py

```diff
@@ -1259,6 +1260,9 @@ def version_add_file(request, addon_id, addon, version_id):
             # Passed validation: sign automatically without manual review.
             new_file.update(status=amo.STATUS_LITE)
             sign_file(new_file)
+    elif new_file_form.cleaned_data.get('beta'):
```

Comment:
At the moment, nothing prevents a fully reviewed unlisted addon from having beta versions, so we need to either:
1/ prevent it (not displaying the "this is a beta" checkbox in the submission form, not detecting it as beta)
2/ allow it, but then also deal with the case where the beta version is for an unlisted addon